src/cli/tc.rs

```diff
@@ -55,53 +56,30 @@ pub fn run() -> Result<()> {
 
 pub async fn init(file_path: &str) -> Result<()> {
   let tailcallrc: resource::Resource<str> = resource_str!("examples/.tailcallrc.graphql");
+  let tailcallrc_path = Path::new(file_path).join(".tailcallrc.graphql");
 
-  let ans = Confirm::new("Do you want to add a file to the project?")
-    .with_default(false)
-    .prompt();
-
-  match ans {
-    Ok(true) => {
-      let file_name = inquire::Text::new("Enter the file name:")
-        .with_default(".graphql")
-        .prompt()
-        .unwrap_or_else(|_| String::from(".graphql"));
-
-      let file_name = format!("{}.graphql", file_name.strip_suffix(".graphql").unwrap_or(&file_name));
-
-      let confirm = Confirm::new(&format!("Do you want to create the file {}?", file_name))
-        .with_default(false)
-        .prompt();
+  if let Some(parent) = tailcallrc_path.parent() {
+    fs::create_dir_all(parent)?;
+  }
 
-      match confirm {
-        Ok(true) => {
-          fs::write(format!("{}/{}", file_path, &file_name), "")?;
+  if tailcallrc_path.exists() {
+    let overwrite = Confirm::new("A .tailcallrc.graphql file already exists. Do you want to overwrite it?")
+      .with_default(false)
+      .prompt()?;
 
-          let graphqlrc = format!(
-            r#"|schema:
-               |- './{}'
-               |- './.tailcallrc.graphql'
-          "#,
-            &file_name
-          )
-          .strip_margin();
-          fs::write(format!("{}/.graphqlrc.yml", file_path), graphqlrc)?;
-        }
-        Ok(false) => (),
-        Err(e) => return Err(e.into()),
-      }
+    if !overwrite {
+      return Ok(());
```

Comment:
We don't need `./graphqlrc.yml` anymore?

Reply:
Yeah, If Im not wrong that's what I got from the issue conversation.

Reply:
@ologbonowiwi We need the file. We don't so much of ceremony around creating the necessary files.

Reply:
In this case, can you re-add it, @neo773?

Reply:
Added